ui/ui-expenses/src/screens/list/list.reactions.js

```diff
@@ -1,9 +1,16 @@
-import pica from 'pica/dist/pica'
-
 export const load = (action, store, { http }) => {
   http('EXPENSES').get('/api/expenses')
 }
 
+export const postImage = async ({ payload }, store, { http, pica }) => {
+  // compress and resize by 2 the image
+  const imageReduced = await pica.reduce(payload, 2)
```

Comment:
Why do you need this second argument here ?

Reply:
This driver is not intended to be reused, so simplify the API if you can

Reply:
In my mind it's better that this is the business code define coeff instead of driver, but ok